zerver/middleware.py

```diff
@@ -96,7 +96,7 @@ def write_log_line(log_data, path, method, remote_ip, email, client_name,
         # Remove non-ascii chars from path (there should be none, if there are it's
         # because someone manually entered a nonexistant path), as UTF-8 chars make
         # statsd sad when it sends the key name over the socket
-        statsd_path = statsd_path.encode('ascii', errors='ignore').decode("ascii")
+        statsd_path = statsd_path.encode('ascii', errors='ignore').decode("ascii") # type: ignore # https://github.com/python/mypy/issues/1540
```

Comment:
This got fixed today. @timabbott, What should we do when this happens? Should we upgrade our requirements.txt to the latest version of mypy, or should we just mention in the comment that this issue is fixed upstream and we should remove the `type: ignore` whenever we upgrade our requirements.txt?